### PR TITLE
[13.0][MIG] sale_order_line_note

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration

### CODE REVIEW

**Comment by dalonsod** [comment]
@sebastienbeau @danimv5 could you review? Thanks!

File: sale_order_line_note/__manifest__.py [hunk 1]
```diff
@@ -6,16 +6,17 @@
...
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Akretion,Odoo Community Association (OCA)",
     "license": "AGPL-3",
+    "maintainers": ["sebastienbeau"],
     "application": False,
     "installable": True,
-    "external_dependencies": {"python": [], "bin": [],},
-    "depends": ["sale",],
-    "data": ["views/sale_order_view.xml",],
+    "external_dependencies": {"python": [], "bin": []},
```
Comment (rousseldenis) [inline sale_order_line_note/__manifest__.py]: Please remove void entries

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Pull request closed without merging**